src/main/java/frc/trigon/robot/commands/factories/AutonomousCommands.java

```diff
@@ -50,15 +50,15 @@ public static Command geAutonomousFeedNoteCommand() {
 
     public static Command getAlignToSpeakerCommand() {
         return new ExecuteEndCommand(
-                () -> getOverrideRotationCommand(Optional.of(SHOOTING_CALCULATIONS.getTargetShootingState().targetRobotAngle().get())).schedule(),
+                () -> getOverrideRotationCommand(Optional.of(SHOOTING_CALCULATIONS.getTargetShootingState().targetRobotAngle().get())).withTimeout(0.2).schedule(),
```

Comment:
You added it to the wrong one and added the wrong thing. Right now, the aiming will stop after a short time. You want the aiming to continue after the note feeds for a short time.